airflow/models/dag.py

```diff
@@ -55,9 +55,6 @@
 from airflow.utils.sqlalchemy import Interval, UtcDateTime
 from airflow.utils.state import State
 
-if TYPE_CHECKING:
```

Comment:
hmm. I like the change but was also happy with the `TYPE_CHECKING` check as we don't need to import those modules otherwise at runtime. The can add unnecessary delays! 

From https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING:

>A special constant that is assumed to be True by 3rd party static type checkers. It is False at runtime. Usage:

Reply:
Well - it will be loaded sooner or later anyway - DAG won't run without BaseOperator loaded. So it's just one extra dictionary lookup here.

Reply:
👍